src/Products/PageTemplates/tests/secure.pt

```diff
@@ -1,5 +1,5 @@
 <div xmlns="http://www.w3.org/1999/xhtml"
      xmlns:tal="http://xml.zope.org/namespaces/tal">
   <span tal:define="soup view/tagsoup | options/soup"
-        tal:replace="structure python: modules['cgi'].escape(soup)" />
+        tal:replace="structure python: modules['cgi'].escape(soup, True)" />
```

Comment:
We should. But it's still there in Python 3.6, so it's not required for porting.

I couldn't remember how to write a conditional import or the equivalent thereof inside a restricted page template. IIRC someone has to declare the cgi (or html) module and the escape function as safe to use as well.